### PR TITLE
Integrate scheduler run and show results on xquant

### PR DESCRIPTION
**Pull request by reuvenperetz**
## Pull Request Description:
Integrate scheduler run and show results on xquant.
Add a flag in DebugConfig to run scheduler for searching minimal max-cut, and save it in the model's metadata. This information can be displayed then in xquant graph.

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tests/keras_tests/feature_networks_tests/feature_networks/compute_max_cut_test.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from mct_quantizers.keras.metadata import get_metadata
+from model_compression_toolkit.constants import TENSORFLOW
+from model_compression_toolkit.target_platform_capabilities.constants import IMX500_TP_MODEL
+from tests.keras_tests.feature_networks_tests.base_keras_feature_test import BaseKerasFeatureNetworkTest
+
+keras = tf.keras
+layers = keras.layers
+
+
+class ComputeMaxCutTest(BaseKerasFeatureNetworkTest):
```
Comment (eladc-git) [inline tests/keras_tests/feature_networks_tests/feature_networks/compute_max_cut_test.py:28]: same test for pytorch?

**Review by eladc-git — CHANGES_REQUESTED** [review]

**Review by eladc-git — APPROVED** [review]
